lib/unhangout-sockets.js

```diff
@@ -169,17 +169,20 @@ _.extend(UnhangoutSocketManager.prototype, events.EventEmitter.prototype, {
 
         mgr.on("create-session", _.bind(function(socket, args) {
             var event = this.getEvent(args.roomId);
-            if (event && socket.user && socket.user.isAdminOf(event)) {
+            if (event && socket.user) {
                 var joinCap = parseInt(args.joinCap ||
                                        models.ServerSession.prototype.MAX_ATTENDEES);
                 if (isNaN(joinCap) || joinCap < 2 || joinCap > 10) {
                     return mgr.writeErr(socket, "Invalid joinCap value");
                 }
+
                 var newSession = new models.ServerSession({
                     title: args.title,
+                    proposedBy: socket.user,
```

Comment:
Now, we are using socket.user.toClientJSON for proposedBy field which not only filters out fields that we consider private but also fixes the problem of a huge chunk of data getting dumped to the console when proposing a session. According to the current implementation, we are not allowing users to change their picture or display name while proposing sessions, which probably means we don't need to make sure that needed fields are included in the data shipped to the client with sessions? In the following commit https://github.com/drewww/unhangout/commit/6e9f755e946022f2006684273187042e3888ee0b, I have added changes which ensure that participant details (user and picture) are shown only if the proposedBy field is not empty. This resolves the issues where old events that didn't have a proposedBy field were not showing aboutnav, event settings etc.